packages/@aws-cdk/custom-resource-handlers/test/aws-s3/notifications-resource-handler/test_index.py

```diff
@@ -95,8 +95,8 @@ def test_create(self, _, put: MagicMock):
         index.handler(event, {})
 
         put.assert_called_once_with(
-            event["ResourceProperties"]["BucketName"],
-            event["ResourceProperties"]["NotificationConfiguration"],
+            Bucket=event["ResourceProperties"]["BucketName"],
```

Comment:
Need to update this test file because this [PR](https://github.com/aws/aws-cdk/commit/37be7b9b5d2abe04aa5f632a7b78aed3e9c233d8) only updated the handler file but didn't change or run the test files. This results in all these tests failing.